views/index.ejs

```diff
@@ -8,10 +8,14 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length %></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
-
+        <% cart.forEach(function (cartItem){ %>
+        <li>
+           <%= cartItem.name %>
+        </li>
+        <% }) %>
```

Comment:
Inside that form you'll want to keep the cartItem's name, but also have a way to access the id so your post route can grab it from req.body.id. Remember how we created hidden inputs with values set to the id so it wouldn't necessarily print to the screen? As a shopper I probably don't care what ID is associated with my cookie. 

`<input type="hidden" name="id" value="<%= cartItem.id %>">`